inbm-lib/inbm_common_lib/constants.py

```diff
@@ -15,6 +15,15 @@
 TEMP_EXT_FOLDER = "/var/cache/manageability/repository-tool/temp_ext"
 URL_NULL_CHAR = '%00'
 
+# Command prefix to run a command 'as the host' using docker, chroot, and namespace control
+# note this will not propagate proxy environment variables
+# change above comment if -e entries are added for proxies
+DOCKER_CHROOT_PREFIX = "/usr/bin/docker run -e DEBIAN_FRONTEND=noninteractive --privileged --rm --net=host --pid=host -v /:/host ubuntu:20.04 /usr/sbin/chroot /host "
+
```

Comment:
Will these constants be used in BC also that it's in the inbm_common_lib instead of the inbm_lib?  I didn't see any changes to BC using either of these constants at the moment.